integration_test/tests/common/mod.rs

```diff
@@ -464,7 +469,7 @@ pub unsafe fn exec_direct_default_query(stmt_handle: HStmt) {
     query.push(0);
     assert_eq!(
         SqlReturn::SUCCESS,
-        SQLExecDirectW(stmt_handle, query.as_ptr(), SQL_NTS as i32),
```

Comment:
There's another SQL_NTS as i32 at line 216 to change.  It needed a `try_into().unwrap()` though.

Reply:
We're going to allow the as conversions in tests to make our life a bit less onerous.